src/main/java/frc/robot/Constants.java

```diff
@@ -14,7 +14,19 @@
 public final class Constants {
 
     public static final class ShooterSpeed{
-        public static final double kShooterSpeed = 0.1;
+        public static final double kShooterSpeed = 0.5;
```

Comment:
These are good data points to have in the `Constants.java` file; but we don't need to create an individual class for each one. We can group the constants in a few different ways -- I'd suggest grouping them by the purpose they serve. 

For instance, we can just have, say,

`public static final class ShooterConstants{}`
`public static final class IntakeConstants{}`

and then put all of the data dealing with the Shooter in `ShooterConstants` and all of the intake data in `IntakeConstants`

This saves us from having to make so many extra classes. It helps with maintenance, and is more memory efficient.

Reply:
👍